app/dashboard/models.py

```diff
@@ -1333,25 +1333,53 @@ class BountyFulfillment(SuperModel):
         ('done', 'done'),
     ]
 
-    fulfiller_address = models.CharField(max_length=50)
+    PAYOUT_TYPE = [
+        ('bounties_network', 'bounties_network'),
+        ('qr', 'qr'),
+        ('fiat', 'fiat')
+    ]
+
+    bounty = models.ForeignKey(Bounty, related_name='fulfillments', on_delete=models.CASCADE, help_text="the bounty against which the fulfillment is made")
+
+    # TODO: RENAME
+    fulfillment_id = models.IntegerField(null=True, blank=True, help_text="bounty's fulfillment number")
+
+    # TODO: RETIRE
+    fulfiller_metadata = JSONField(default=dict, blank=True)
+
+    fulfiller_address = models.CharField(max_length=50, help_text="address to which amount is credited")
+    funder_address = models.CharField(max_length=50, null=True, help_text="address from which amount is deducted")
+
+    # TODO: rename to fulfiller_profile
+    profile = models.ForeignKey('dashboard.Profile', related_name='fulfilled', on_delete=models.CASCADE, null=True, help_text="fulfillers's profile")
+    funder_profile = models.ForeignKey('dashboard.Profile', null=True, on_delete=models.CASCADE, help_text="funder's profile")
+
+    # TODO: MAKE AS PROP
     fulfiller_email = models.CharField(max_length=255, blank=True)
+    # TODO: MAKE AS PROP
     fulfiller_github_username = models.CharField(max_length=255, blank=True)
+    # TODO: MAKE AS PROP
     fulfiller_name = models.CharField(max_length=255, blank=True)
-    fulfiller_metadata = JSONField(default=dict, blank=True)
-    fulfillment_id = models.IntegerField(null=True, blank=True)
+
+    # TODO: rename to hours_worked
     fulfiller_hours_worked = models.DecimalField(null=True, blank=True, decimal_places=2, max_digits=50)
+    # TODO: rename to submission_url
     fulfiller_github_url = models.CharField(max_length=255, blank=True, null=True)
     funder_last_notified_on = models.DateTimeField(null=True, blank=True)
-    accepted = models.BooleanField(default=False)
-    accepted_on = models.DateTimeField(null=True, blank=True)
 
-    bounty = models.ForeignKey(Bounty, related_name='fulfillments', on_delete=models.CASCADE)
-    profile = models.ForeignKey('dashboard.Profile', related_name='fulfilled', on_delete=models.CASCADE, null=True)
+    accepted = models.BooleanField(default=False, help_text="has the fulfillment been accepted by the funder")
+    accepted_on = models.DateTimeField(null=True, blank=True, help_text="date when the fulfillment was accepted by the funder")
+
+    payout_type = models.CharField(max_length=20, null=True, blank=True,choices=PAYOUT_TYPE, help_text="payment type used to make the payment")
+    tenant = models.CharField(max_length=10, null=True, blank=True, help_text="specific tenant type under the payout_type")
```

Comment:
choices here as well? what do you think?

Reply:
tenant would be super long list cause of the standard bounties 
For every ERC-20 token -> tenant == token_name // This is along list
For cross chain -> tenant == token_name except cUSD / cGLD
For fiat -> tenant = PYPL